ci/py311_simplification-latest.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   - esda
   - geopandas
   - graphblas-algorithms
-  - h3-py
+  - h3-py=3.7.7
```

Comment:
I even think that now this only contains code for the paper, we should freeze the env completely with pixi. But we can talk about that on Thursday.